queries.sql

```diff
@@ -110,3 +106,81 @@ JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.full_name
 ORDER BY animal_count DESC
 LIMIT 1;
+
+-- Query Multiple Tables;
+SELECT a.name
+FROM animals a
+JOIN visits v ON a.id = v.animals_id
+JOIN vets vt ON v.vets_id = vt.id
+WHERE vt.name = 'William Tatcher'
+ORDER BY v.visiting_date DESC
+LIMIT 1;
+
+
+SELECT COUNT(DISTINCT v.animals_id) AS animal_count
+FROM visits v
+JOIN vets vt ON v.vets_id = vt.id
+WHERE vt.name = 'Stephanie Mendez';
+
+
+SELECT vt.name, s.name
+FROM vets vt
+LEFT JOIN specializations sp ON vt.id = sp.vets_id
+LEFT JOIN species s ON sp.species_id = s.id;
+
+
+SELECT a.name
+FROM animals a
+JOIN visits v ON a.id = v.animals_id
+JOIN vets vt ON v.vets_id = vt.id
+WHERE vt.name = 'Stephanie Mendez'
+  AND v.visiting_date >= '2020-04-01'
+  AND v.visiting_date <= '2020-08-30';
+
+
+SELECT a.name, COUNT(*) AS visit_count
+FROM animals a
+JOIN visits v ON a.id = v.animals_id
+GROUP BY a.name
+ORDER BY visit_count DESC
+LIMIT 1;
+
+
+
+SELECT a.name, MIN(v.visiting_date) AS first_visit
+FROM animals a
+JOIN visits v ON a.id = v.animals_id
+JOIN vets vt ON v.vets_id = vt.id
+WHERE vt.name = 'Maisy Smith'
+GROUP BY a.name
+ORDER BY first_visit ASC
+LIMIT 1;
+
+
+SELECT a.name AS animal_name, v.visiting_date, vt.name AS vet_name, vt.age AS vet_age
+FROM animals a
+JOIN visits v ON a.id = v.animals_id
+JOIN vets vt ON v.vets_id = vt.id
+WHERE v.visiting_date = (
+    SELECT MAX(visiting_date)
+    FROM visits
+)
+LIMIT 1;
+
+
+SELECT COUNT(*) AS num_visits
+FROM visits v
+JOIN vets vt ON v.vets_id = vt.id
+JOIN animals a ON v.animals_id = a.id
+JOIN specializations s ON vt.id = s.vets_id AND a.id = s.species_id
+WHERE s.species_id IS NULL;
+
+
+SELECT a.name, COUNT(*) AS num_visits
+FROM visits v
+JOIN animals a ON v.animals_id = a.id
+JOIN vets vt ON v.vets_id = vt.id
+WHERE vt.name = 'Maisy Smith'
+GROUP BY a.name
+ORDER BY num_visits DESC
+LIMIT 1;
```

Comment:
- [ ] Lastly, this query is also supposed to return the specialty that  Maisy Smith should consider getting. You can look for the species she gets the most to determine this. But from the screenshot you provided which I have pasted below, your query is returning `Boarmon` which isn't a specialty but an animal :+1: 
- [ ] So your query should return either `Digimon` or `Pokemon` :wink: 
- Kindly fix this as well and replace those queries in your PR with the right screenshots :+1: 
![image](https://github.com/joyapisi/vet-clinic-db/assets/102699958/08aa0189-89fe-402b-85a5-636b19e95bc7)